gulpfile.js

```diff
@@ -3,153 +3,154 @@
-				'**/*.html',
-				'**/*.css',
-				'**/*.js',
-				'**/*.svg',
-				'**/*.ttf',
-				'**/*.otf',
-				'**/*.eot',
-				'**/*.woff',
-				'**/*.woff2',
-
-				// include specific files and folders
-				'screenshot.png',
-
-				// exclude files and folders
-				'!node_modules/**/*',
-				'!assets/bower_components/**/*',
-				'!style.css.map',
-				'!assets/js/custom/*',
-				'!assets/css/patrials/*'
-
-			];
+var project = 'neat', // Project name, used for build zip.
+    url             = 'neat.dev', // Local Development URL for BrowserSync. Change as-needed.
+    bower           = './assets/bower_components/'; // Not truly using this yet, more or less playing right now. TO-DO Place in Dev branch
+    build           = './buildtheme/', // Files that you want to package into a zip go here
+    buildInclude    = [
+        // include common file types
+        '**/*.php',
+        '**/*.html',
+        '**/*.css',
+        '**/*.js',
+        '**/*.svg',
+        '**/*.ttf',
+        '**/*.otf',
+        '**/*.eot',
+        '**/*.woff',
+        '**/*.woff2',
+
+        // include specific files and folders
+        'screenshot.png',
+
+        // exclude files and folders
+        '!node_modules/**/*',
+        '!assets/bower_components/**/*',
+        '!style.css.map',
+        '!assets/js/custom/*',
+        '!assets/css/partials/*'
+    ],
+    // Optional FTP connection information (do not check this in)
+    ftpInfo = {
+        host:     'ftp.mysite.com',
+        user:     'my@user.name',
+        password: 'mypass',
+        parallel: 5 // Max # of parallel connections
+    };
 
 // Load plugins
-	var gulp     = require('gulp'),
-		browserSync  = require('browser-sync'), // Asynchronous browser loading on .scss file changes
-		reload       = browserSync.reload,
-		autoprefixer = require('gulp-autoprefixer'), // Autoprefixing magic
-		minifycss    = require('gulp-uglifycss'),
-		filter       = require('gulp-filter'),
-		uglify       = require('gulp-uglify'),
-		imagemin     = require('gulp-imagemin'),
-		newer        = require('gulp-newer'),
-		rename       = require('gulp-rename'),
-		concat       = require('gulp-concat'),
-		notify       = require('gulp-notify'),
-		cmq          = require('gulp-combine-media-queries'),
-		runSequence  = require('gulp-run-sequence'),
-		sass         = require('gulp-sass'),
-		plugins      = require('gulp-load-plugins')({ camelize: true }),
-		ignore       = require('gulp-ignore'), // Helps with ignoring files and directories in our run tasks
-		rimraf       = require('gulp-rimraf'), // Helps with removing files and directories in our run tasks
-		zip          = require('gulp-zip'), // Using to zip up our packaged theme into a tasty zip file that can be installed in WordPress!
-		plumber      = require('gulp-plumber'), // Helps prevent stream crashing on errors
-		cache        = require('gulp-cache'),
-		sourcemaps   = require('gulp-sourcemaps');
-
+var $ = require('gulp-load-plugins')();
```

Comment:
Understandable. I personally prefer using the single dollar sign instead of managing variable names. Choose what you want to take from my changes :)

Thanks for setting up a great workflow, by the way!